lasagne/layers/recurrent.py

```diff
@@ -1005,10 +1065,11 @@ def get_output_for(self, input, mask=None, *args, **kwargs):
         input = input.dimshuffle(1, 0, 2)
         seq_len, num_batch, _ = input.shape
 
-        # precompute inputs*W and dimshuffle
-        # W_in is (n_features, 3*num_units). input_dot_W is then
-        # (n_batch, n_time_steps, 3*num_units).
-        input_dot_W = T.dot(input, self.W_in_stacked) + self.b_stacked
+        if self.precompute_input:
+            # precompute_input inputs*W and dimshuffle
```

Comment:
Can you update this comment?  We aren't dimshuffling here anymore, and `W_in` is called `W_in_stacked`.